ceph-grafana/build/build

```diff
@@ -0,0 +1,5 @@
+sudo yum install -y buildah
```

Comment:
Is `buildah` just a wrapper like `podman-docker`?  Is it going to cause conflicts with other jobs that install podman/docker packages?

Reply:
No, just another tool that uses some of the same libraries as podman.  It builds container images.